app/helpers/alchemy/admin/essences_helper.rb

```diff
@@ -9,6 +9,17 @@ module EssencesHelper
       # Renders the Content editor partial from the given Content.
       # For options see -> render_essence
       def render_essence_editor(content, options = {}, html_options = {})
+        if !options.empty?
+          Alchemy::Deprecation.warn <<~WARN
```

Comment:
Given that this simply doesn't do anything any longer, I think this should be an error.

Reply:
I will extract the deprecations into the 4.3 release.